utopia-vscode-extension/src/extension.ts

```diff
@@ -316,6 +317,92 @@ function sendMessageToUtopia(message: FromVSCodeToUtopiaMessage): void {
   vscode.commands.executeCommand('utopia.toUtopiaMessage', message)
 }
 
+function handleFromUtopiaToVSCodeMessage(
```

Comment:
This is just extracting an inner anonymous function into its own top level function, as requested in the previous PR